pkgs/tools/networking/photon/default.nix

```diff
@@ -1,32 +1,27 @@
-{ stdenv, pythonPackages, fetchurl, makeWrapper }:
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper }:
 
-with pythonPackages;
-buildPythonApplication rec {
-  pname = "photon";
-  version = "1.0.7";
+python3Packages.buildPythonPackage rec {
```

Comment:
Is photon now meant to be used as a library?

If not, please keep using `buildPythonApplication`.

Reply:
no but i need python3.

Reply:
And why can't you use `pythonPackages.buildPythonApplication`?

Reply:
i did finally. was a mistake.